Jenkinsfile

```
@@ -220,22 +230,29 @@ pipeline {
 								runOnNativeBuildAgent("${PLATFORM}") {
 									cleanWs() // Workspace is not cleaned up by default, so we do it explicitly
 									echo "OS: ${os}, ARCH: ${arch}"
-									unstash "swt.binaries.sources.${ws}"
+									unstash "swt.binaries.sources.${ws == 'gtk4' ? 'gtk' : ws }"
 									dir('jdk.resources') {
 										unstash "jdk.resources.${os}.${arch}"
 									}
 									withEnv(['MODEL=' + arch, "OUTPUT_DIR=${WORKSPACE}/libs", "SWT_JAVA_HOME=${WORKSPACE}/jdk.resources"]) {
 										if (isUnix()){
 											sh '''
```

As the script below that runs on the native machine now uses bash specific commands (`[[`) and bash seems not to be the default shell on all native machines, we have to add a shebang. 
```suggestion
											sh '''#!/bin/bash -x
```

---

:+1: that makes sense, the block further down that relies on bash runs on the main build machine. I will include the shebang on it too.

---

> the block further down that relies on bash runs on the main build machine

Yes it does. The main build machines use bash by default. Altough there has been some trouble during the transition from centos images to Ubunut images because initially `sh` was the default. So having some more explicit here will not hurt.